pkg/transform/registries_transform.go

```diff
@@ -0,0 +1,86 @@
+package transform
+
+import (
+	"github.com/BurntSushi/toml"
+	"github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v2"
+)
+
+type RegistriesExtraction struct {
+	Registries map[string]Registries
+}
+
+type Registries struct {
+	List []string `toml:"registries"`
+}
+
+type ImageCR struct {
+	APIVersion string   `yaml:"apiVersion"`
+	Kind       string   `yaml:"kind"`
+	Metadata   Metadata `yaml:"metadata"`
+	Spec       struct {
```

Comment:
[BLOCKER] Shoudln't inline this type. It's a pain in the ass, but I think they should be broken out into their own types.

Reply:
Own type but it's okay in the same file.

Reply:
Removed this here, sdn_transform.go, and pkg/oauth/oauth.go. I think I got them all.